docs/CONTRIBUTING.md

```diff
@@ -35,6 +35,58 @@ The source code of ABACUS is based on several modules. Under the ABACUS root dir
+|   |-- module_neighbor         The module for finding the neighbors of each atom in the unit cell.
+|   |-- module_paw              The module for performing PAW calculations.
+|   |-- module_symmetry         The module for finding the symmetry operations of the unit cell.
+|-- module_elecstate            The module for defining the electronic state and its operations.
+|   |-- module_charge           The module for calculating the charge density, charge mixing
+|   |-- potentials              The module for calculating the potentials, including Hartree, exchange-correlation, local pseudopotential, etc.
+|-- module_esolver              The module for solving the eigenvalue problem with different theories including Kohn-Sham DFT, Stochastic DFT,
+|   |                           TDDFT, Orbital-free DFT, etc.
+|-- module_hamilt_general       The module for defining general Hamiltonian that can be used both in PW and LCAO calculations. 
+|   |-- module_ewald            The module for calculating the Ewald summation.
+|   |-- module_surchem          The module for calculating the surface charge correction.
+|   |-- module_vdw              The module for calculating the van der Waals correction.
+|   |-- module_xc               The module for calculating the exchange-correlation energy and potential.
+|-- module_hamilt_lcao          The module for defining the Hamiltonian in LCAO calculations.
+|   |-- hamilt_lcaodft          The module for defining the Hamiltonian in LCAO-DFT calculations.
+|   |   |-- operator_lcao       The module for defining the operators in LCAO-DFT calculations.
+|   |-- module_deepks           The module for defining the Hamiltonian in DeepKS calculations.
+|   |-- module_dftu             The module for defining the Hamiltonian in DFT+U calculations.
+|   |-- module_gint             The module for performing grid integral in LCAO calculations.
+|   |-- module_hcontainer       The module for storing the Hamiltonian matrix in LCAO calculations.
+|   `-- module_tddft            The module for defining the Hamiltonian in TDDFT calculations.
+|-- module_hamilt_pw            The module for defining the Hamiltonian in PW calculations.
+|   |-- hamilt_ofdft            The module for defining the Hamiltonian in OFDFT calculations.
+|   |-- hamilt_pwdft            The module for defining the Hamiltonian in PW-DFT calculations.
+|   |   |-- operator_pw         The module for defining the operators in PW-DFT calculations.
+|   `-- hamilt_stodft           The module for defining the Hamiltonian in STODFT calculations.
+|-- module_hsolver              The module for solving the Hamiltonian with different diagonalization methods, including CG, Davidson in PW 
+|   |                           calculations, and scalapack and genelpa in LCAO calculations.
+|-- module_io                   The module for output properties including band structure, density of states, charge density, etc.
```

Comment:
module_io also reads INPUT, INPUTw files. So it is actually what should be refactorized in the future: now that it is named as "io", the reading of KPT file and STRU, orb and upf file are implemented in UnitCell class which is in module_cell module. So temporarily I would suggest add "reading of INPUT files" ahead of the present description.

Reply:
Thanks for you valuable suggestions. I will add the description!